plugins/plugins-api/src/main/java/org/codice/compliance/saml/plugin/IdpPostResponse.java

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.compliance.saml.plugin;
+
+import static org.apache.cxf.rs.security.saml.sso.SSOConstants.RELAY_STATE;
+import static org.apache.cxf.rs.security.saml.sso.SSOConstants.SAML_RESPONSE;
+
+import com.jayway.restassured.path.xml.element.Node;
+
+/**
+ * This class is the return type for methods of the {@code IdpResponder} interface for the POST
+ * Binding. An internal static builder class {@code IdpPostResponseBuilder} should be used to build
+ * the {@code IdpPostResponse} object.
+ *
+ * <p>The implemented {@code IdpResponder} methods should call the builder methods:
+ *
+ * <ul>
+ *   <li>IdpPostResponseBuilder.httpStatusCode(int)
+ *   <li>IdpPostResponseBuilder.samlForm(Node)
+ * </ul>
+ *
+ * Before building the {@code IdpPostResponse} object.
+ *
+ * <p>Example usage:
+ *
+ * <p>
+ *
+ * <blockquote>
+ *
+ * <pre>
+ *   IdpPostResponse.IdpPostResponseBuilder idpPostResponseBuilder = new IdpPostResponse.IdpPostResponseBuilder();
+ *   idpPostResponseBuilder.httpStatusCode(exampleStatusCode)
+ *       .samlForm(exampleSamlForm);
+ *   return idpPostResponseBuilder.build();
+ * </pre>
+ *
+ * </blockquote>
+ */
+public class IdpPostResponse {
+  private IdpPostResponse() {}
+
+  public static class IdpPostResponseBuilder {
```

Comment:
✏️ You can just name this `Builder`. Same for the sibling class. Then they're accessed like `new IdpPostResponse.Builder()`.